src/modules/Crop.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Image Cropping module
+ */
+ module.exports = function Crop(options){
+
+   options = options || {};
+   options.title = "Crop Image";
+   options.format = options.format || "png";
+
+   function draw(image) {
+     var getPixels = require("get-pixels"),
+         savePixels = require("save-pixels"),
+         base64 = require('base64-stream');
+
+     getPixels(image.src,function(err,pixels){
+       var newdata = [];
+       var ox = options.cropX || 0; //Where to begin the crop on X axis
+       var oy = options.cropY || 0; //Where to begin the crop on Y axis
+       var w = options.cropL || Math.floor(0.5*pixels.shape[0]); //Length of crop
+       var h = options.cropH || Math.floor(0.5*pixels.shape[1]); //Height of crop
+       var iw = pixels.shape[0]; //Width of Original Image
+       newarray = new Uint8Array(4*w*h);
+       for (var n = oy; n < oy + h; n++) {
+         newarray.set(pixels.data.slice(n*4*iw + ox, n*4*iw + ox + 4*w),4*w*(n-oy));
+       }
+       pixels.data = newarray;
+       pixels.shape = [w,h,4];
+       pixels.stride[1] = 4*w;
+
+       var buffer = base64.encode();
+       savePixels(pixels, options.format)
+         .on('end', function() {
+
+         var img = new Image();
+
+         img.src = 'data:image/' + options.format + ';base64,' + buffer.read().toString();
```

Comment:
I think we should try to abstract this to make it easier to do from different modules, but will merge for now. Thanks!